### ISSUE TITLE
NOD missions build 3 sam sites

### ISSUE THREAD

**Issue by Nineeva** [OpenRA/OpenRA#12215]
A lot of the NOD missions have a secondary objective 'build 3 SAMs to fend off the GDI bombers' though when you build them it doesn't stop the bombers.  From what I remember from the original game  they would just try shoot them down.  Since they can't be shot down could it just have a trigger to stop them altogether once you build 3?

**Comment by abcdefg30** [comment]
The bombers will just avoid bombing the areas covered by SAMs.

Maybe we should try to make that more clear (or rework the missions, like you suggested, to allow shooting them down or entirely disable them after three SAMs have been built).

**Comment by Nineeva** [comment]
Yeah I see, it works quite well just making the bombers avoid the area, but would be nice to have a better description like you say as I wouldn't have know without you telling me.

**Comment by abcdefg30** [comment]
I think a good idea here might be making A10 Bombers targetable by SAMs (and other AA) which is actually suggested as well in #16624.
